homeassistant/components/bmw_connected_drive/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/bmw_connected_drive",
   "iot_class": "cloud_polling",
   "loggers": ["bimmer_connected"],
-  "requirements": ["bimmer_connected==0.12.1"]
+  "requirements": ["bimmer-connected==0.13.0"]
```

Comment:
The dependency bump should be a standalone PR

Reply:
Moved dependency bump + related test changes to #90127.